artifactory/commands/pip/install.go

```diff
@@ -201,7 +201,9 @@ func getPackageName(pythonExecutablePath string, pipArgs []string) (string, erro
 	// Extract package name from setup.py.
 	packageName, err := piputils.ExtractPackageNameFromSetupPy(filePath, pythonExecutablePath)
 	if err != nil {
-		return "", errors.New("Failed determining module-name from 'setup.py' file: " + err.Error())
+		// If setup.py egg_info command failed we use build name as module name and continue to pip-install execution
+		log.Info("Failed determining module-name by running 'setup.py egg_info' command: " + err.Error())
```

Comment:
What do you think of a little less frightening syntax? This info shouldn't concern the users much.
```suggestion
		log.Info("Couldn't determine module-name after running the 'egg_info' command: " + err.Error())
```